packages/core/src/with-router.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import {withRouter as withRouterNext, NextRouter as NextRouterNext} from 'next/router'
```

Comment:
You only need 'next' one time in `NextRouterNext` :D

Reply:
Admittedly, this renaming is unnecessary.
Fixed.